rep-0145.rst

```diff
@@ -0,0 +1,190 @@
+REP: 145
```

Comment:
Hi @paulbovbel , I hope you do not mind me adding a few comments here. I am not an expert, just take these as noisy data points!